sdk/core/azure-core/src/main/java/com/azure/core/util/logging/LoggingEventBuilder.java

```diff
@@ -250,7 +251,7 @@ public Throwable log(Throwable throwable) {
         Objects.requireNonNull(throwable, "'throwable' cannot be null.");
 
         if (this.isEnabled) {
-            performLogging(level, null, throwable);
+            performLogging(level, getMessageWithContext(null, throwable), logger.isDebugEnabled() ? throwable : null);
```

Comment:
same - what to do with an exception object is consumer, not producer decision

Reply:
Same comment as the other case